scripts/modisSuite/Downloader.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/python3
+# coding: utf8
+"""
+Auteur: Gabriel Desharnais
```

Comment:
What were the extent of the changes you made to Gabriel's code? Ideally we want to push those changes back to Garbriel so his module contains the necessary fixes.